test/jdk/java/awt/regtesthelpers/Util.java

```diff
@@ -318,12 +318,20 @@ public static void drag(Robot robot, Point startPoint, Point endPoint, int butto
             throw new IllegalArgumentException("invalid mouse button");
         }
 
+        boolean autoDelaySet = false;
+        if(robot.getAutoDelay() == 0) {
+            robot.setAutoDelay(100);
+            autoDelaySet = true;
+        }
```

Comment:
If this is the right place to fix it - in the Util class - then you can just call delay(100) in the places you need it - no need to set and then unset auto-delay.

Changing Util you will need to re-validate all tests that use this utility method.

Based on the explanation you gave ~mrserb I wonder if the test should also be waiting for the drop before disposing, not just throwing an arbitrary wait into the mix.

Reply:
I have removed the changes done in Util.java and make the changes in all the 4 test files, so that the test will be waiting for the drop before disposing.